src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,36 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
```

Comment:
Is this correct? Doesn't it have to look at the data to find the mean, variance, min, max etc?

Reply:
It should be yes, thanks for mentioning. I will correct this.

---
In reply to: [276894215](https://github.com/dotnet/machinelearning/pull/3432#discussion_r276894215) [](ancestors = 276894215)